src/Attachment.tsx

```diff
@@ -115,6 +115,21 @@ const Media = (props: {
     }
 }
 
+const Unknown = (props: {
+    format: FormatState,
+    contentType: string,
+    contentUrl: string,
+    name: string
+}) => {
+    if (regExpCard.test(props.contentType)) {
+        return <span>{ props.format.strings.unknownCard.replace('%1', props.contentType) }</span>;
+    } else if (props.contentUrl) {
+        return <span><a href={ props.contentUrl } title={ props.contentUrl } target='_blank'>{ props.name || props.format.strings.unknownFile.replace('%1', props.contentType) }</a></span>;
+    } else {
+        return <span>{ props.format.strings.unknownFile.replace('%1', props.contentType) }</span>;
```

Comment:
looks like maybe there is on more collapse?  I think if you refactored the if... else if... else ... then you can have the one fallback.